src/extension/background-script/actions/setup/reset.ts

```diff
@@ -3,11 +3,11 @@ import type { MessageReset } from "~/types";
 import state from "../../state";
 
 const reset = async (message: MessageReset) => {
+  await state.getState().password(null);
```

Comment:
@im-adithya please check if this is correct. Just adjusted it according to the mv3 way of handling the password.